src/main/java/ch/njol/skript/expressions/ExprMessage.java

```diff
@@ -44,103 +45,119 @@
+			String get(Event event) {
 				if (PlayerChatEventHandler.usesAsyncEvent)
-					return ((AsyncPlayerChatEvent) e).getMessage();
+					return ((AsyncPlayerChatEvent) event).getMessage();
 				else
-					return ((PlayerChatEvent) e).getMessage();
+					return ((PlayerChatEvent) event).getMessage();
 			}
 			
 			@Override
-			void set(final Event e, final String message) {
+			void set(Event event, String message) {
 				if (PlayerChatEventHandler.usesAsyncEvent)
-					((AsyncPlayerChatEvent) e).setMessage(message);
+					((AsyncPlayerChatEvent) event).setMessage(message);
 				else
-					((PlayerChatEvent) e).setMessage(message);
+					((PlayerChatEvent) event).setMessage(message);
 			}
 		},
 		JOIN("join", "(join|log[ ]in)( |-)message", PlayerJoinEvent.class) {
 			@Override
 			@Nullable
-			String get(final Event e) {
-				return ((PlayerJoinEvent) e).getJoinMessage();
+			String get(Event event) {
+				return ((PlayerJoinEvent) event).getJoinMessage();
 			}
 			
 			@Override
-			void set(final Event e, final String message) {
-				((PlayerJoinEvent) e).setJoinMessage(message);
+			void set(Event event, String message) {
+				((PlayerJoinEvent) event).setJoinMessage(message);
 			}
 		},
 		QUIT("quit", "(quit|leave|log[ ]out|kick)( |-)message", PlayerQuitEvent.class, PlayerKickEvent.class) {
 			@Override
 			@Nullable
-			String get(final Event e) {
-				if (e instanceof PlayerKickEvent)
-					return ((PlayerKickEvent) e).getLeaveMessage();
+			String get(Event event) {
+				if (event instanceof PlayerKickEvent)
+					return ((PlayerKickEvent) event).getLeaveMessage();
 				else
-					return ((PlayerQuitEvent) e).getQuitMessage();
+					return ((PlayerQuitEvent) event).getQuitMessage();
 			}
 			
 			@Override
-			void set(final Event e, final String message) {
-				if (e instanceof PlayerKickEvent)
-					((PlayerKickEvent) e).setLeaveMessage(message);
+			void set(Event event, String message) {
+				if (event instanceof PlayerKickEvent)
+					((PlayerKickEvent) event).setLeaveMessage(message);
 				else
-					((PlayerQuitEvent) e).setQuitMessage(message);
+					((PlayerQuitEvent) event).setQuitMessage(message);
 			}
 		},
 		DEATH("death", "death( |-)message", EntityDeathEvent.class) {
 			@Override
 			@Nullable
-			String get(final Event e) {
-				if (e instanceof PlayerDeathEvent)
-					return ((PlayerDeathEvent) e).getDeathMessage();
+			String get(Event event) {
+				if (event instanceof PlayerDeathEvent)
+					return ((PlayerDeathEvent) event).getDeathMessage();
 				return null;
 			}
 			
 			@Override
-			void set(final Event e, final String message) {
-				if (e instanceof PlayerDeathEvent)
-					((PlayerDeathEvent) e).setDeathMessage(message);
+			void set(Event event, String message) {
+				if (event instanceof PlayerDeathEvent)
+					((PlayerDeathEvent) event).setDeathMessage(message);
+			}
+		},
+		UNKNOWN_CMD("unknown command", "unknown( |-)command( |-)message", UnknownCommandEvent.class) {
```

Comment:
This is a Paper exclusive event. Needs a resolution. Probably can't be part of the enum.